### DISCUSSION TITLE
create an organization that belongs to "A business or institution"

### DISCUSSION THREAD

**Discussion by cuixinye** [community/community#63148] [New to GitHub]
I'm an employee of some company. Should I worry about legal issues before I create an organization belongs to company.
I googled relevant information and read Github Docs, but found no clear conclusion. Here's my question.

1. If I've chosen to create a free plan in the previous step, will it ask me or company to provide billing information? 
2. Will you identify the company information and send a verification email and such?
3. Is it acceptable that there are multi free organizations that belongs to one same company?

**Comment by cesu171** [comment]
1. No, you don't need to provide any billing information
2. To verify the organization, you need to verify with domain, and [here](https://docs.github.com/en/organizations/managing-organization-settings/verifying-or-approving-a-domain-for-your-organization) is how you can do it.
3. Yes
